pydle/features/rpl_whoishost/rpl_whoishost.py

```diff
@@ -13,13 +13,14 @@ async def on_raw_378(self, message):
         host = data[-2]
 
         meta = {"real_ip_address": ip_addr, "real_hostname": host}
-        self._sync_user(target, meta)
+        await self._sync_user(target, meta)
         if target in self._whois_info:
             self._whois_info[target]["real_ip_address"] = ip_addr
             self._whois_info[target]["real_hostname"] = host
 
     async def whois(self, nickname):
         info = await super().whois(nickname)
+        if info is None: return info
```

Comment:
(style) please put the return statement on a second line
```suggestion
        if info is None: 
            return info
```